### PR TITLE
"Enabled" WC Payment account status support

### PR DESCRIPTION
**Pull request by kidinov**
The PR adds handling of the "enabled" status.

https://stripe.com/docs/connect/dashboard#enabled

### CODE REVIEW

File: example/src/androidTest/java/org/wordpress/android/fluxc/mocked/MockedStack_WCInPersonPaymentsTest.kt [hunk 1]
```diff
@@ -159,6 +159,15 @@ class MockedStack_InPersonPaymentsTest : MockedStack_Base() {
         assertTrue(result.result?.status == WCPaymentAccountStatus.RESTRICTED_SOON)
     }
 
+    @Test
+    fun whenLoadAccountEnabledStatusThenRestrictedSoonStatusReturned() = runBlocking {
```
Comment (rossanafmenezes) [inline example/src/androidTest/java/org/wordpress/android/fluxc/mocked/MockedStack_WCInPersonPaymentsTest.kt]: ❓ I'm not sure I understand the naming of the test. Aren't we returning the Enabled status here? Shouldn't it be `whenLoadAccountEnabledStatusThenEnabledStatusReturned()`? Pls ignore me if I misunderstood it.

**Review by rossanafmenezes — COMMENTED** [review]
LGTM. Left a quick question about the test naming.

File: example/src/androidTest/java/org/wordpress/android/fluxc/mocked/MockedStack_WCInPersonPaymentsTest.kt (hunk 1 above)
Comment (kidinov) [reply example/src/androidTest/java/org/wordpress/android/fluxc/mocked/MockedStack_WCInPersonPaymentsTest.kt]: Good catch! Fixed - 723477f9cfb3fe388ca8a23a1cb8f67e18308753

**Review by rossanafmenezes — APPROVED** [review]
LGTM :shipit:

**Pull request merged**